gnucash/import-export/import-main-matcher.h

```diff
@@ -36,6 +36,15 @@
 #include "Transaction.h"
 #include "import-backend.h"
 
+/* global variable to flag that the add checkbox has been toggled so that 
+  the view_select_function for the treeview can negate selection of a row
+  immediately after the checkbox has been toggled. Avoids a situation where
+  if one of the other checkboxes is (U+R or R) toggled, the row is left in
+  a permanently selected state and cannot be unselected unless the Add 
+  checkbox is toggled again
+*/
+extern gboolean add_toggled;
```

Comment:
I do not think you need this, not certain but if you add it to info as suggested then you definitely wont.

Reply:
agreed I'll try that out